.dockerignore

```diff
@@ -55,9 +56,8 @@
 !protos/
 
 !bazel/
-!*.bzl
-!*.bazel
-!*.BUILD
+!**/*.bzl
+!**/*.bazel
```

Comment:
In the current file this is only valid for the top-level files. `.BUILD` files exist only in the `bazel` folder.